tests/test_classifier/__init__.py

```diff
@@ -0,0 +1,6 @@
+#Backend must be set before first use. 
+# Setting backend here allows us to run tests just in this folder, without running the whole yellowbrick.tests folder
+# This command will have no effect if backend has already been set previously. 
+import matplotlib
+matplotlib.use('Agg')
+
```

Comment:
Did the [VisualTestCase](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/base.py#L28) not work to ensure that the backend was set correctly? It looks like that's the base class of all the tests, and it was supposed to set the backend on setUp.